pkg/sdk/pipes.go

```diff
@@ -0,0 +1,190 @@
+package sdk
+
+import (
+	"context"
+	"database/sql"
+	"time"
+)
+
+type Pipes interface {
+	// Create creates a pipe.
+	Create(ctx context.Context, id SchemaObjectIdentifier, copyStatement string, opts *PipeCreateOptions) error
+	// Alter modifies an existing pipe.
+	Alter(ctx context.Context, id SchemaObjectIdentifier, opts *PipeAlterOptions) error
+	// Drop removes a pipe.
+	Drop(ctx context.Context, id SchemaObjectIdentifier) error
+	// Show returns a list of pipes.
+	Show(ctx context.Context, opts *PipeShowOptions) ([]*Pipe, error)
+	// ShowByID returns a pipe by ID.
+	ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*Pipe, error)
+	// Describe returns the details of a pipe.
+	Describe(ctx context.Context, id SchemaObjectIdentifier) (*Pipe, error)
+}
+
+// PipeCreateOptions contains options for creating a new pipe in the system for defining the COPY INTO <table> statement
+// used by Snowpipe to load data from an ingestion queue into tables.
```

Comment:
It is only a high level overview which IMO will almost never change. Links should also not change drastically (because what's the reason?).

And I don't get the clickability remark: url is now clickable without using this suggested format.